npm/webpack-preprocessor/package.json

```diff
@@ -34,7 +34,6 @@
     "chalk": "3.0.0",
     "chokidar-cli": "2.1.0",
     "common-tags": "^1.8.2",
-    "cypress": "0.0.0-development",
```

Comment:
Do we need to remove this?

Reply:
I see the `cypress` types being used still in the lib. So I think we just put this back if it doesn't cause a circular dep

Reply:
yeah I'll add them back